packages/account/src/Sections/Profile/LanguageSettings/language-settings.tsx

```diff
@@ -9,21 +9,13 @@ import { Formik, FormikHandlers, FormikHelpers, FormikValues } from 'formik';
 import FormFooter from 'Components/form-footer';
 import LanguageRadioButton from 'Components/language-settings';
 
-type TchangeCurrentLanguage = (lang: string) => void;
-
 type TLanguageSettings = {
     current_language: string;
-    changeCurrentLanguage: TchangeCurrentLanguage;
-    changeLanguage: (lang: string, changeCurrentLanguage: TchangeCurrentLanguage) => Promise<void>;
+    changeSelectedLanguage: (lang: string) => Promise<void>;
```

Comment:
```suggestion
    changeSelectedLanguage: (lang: string) => void;
```
@yashim-deriv `changeSelectedLanguage` is not async if I'm not mistaken, Also no need to `await` for it at line `27` 🤔